example/image-classification/train_model.py

```diff
@@ -73,8 +73,8 @@ def fit(args, network, data_loader, sym_gen=None):
         learning_rate      = args.lr,
         momentum           = 0.9,
         wd                 = 0.00001,
-        initializer        = mx.init.Xavier(factor_type="in", magnitude=2.34),
         sym_gen            = sym_gen,
+        initializer        = mx.init.Xavier(actor_type="in", rnd_type="gaussian", magnitude=2.0),
```

Comment:
@jmschrei  no i mean the rand_type, default is uniform distribution, which may not be right. 
according to this paper http://arxiv.org/pdf/1502.01852v1.pdf  when the active type is relu , we should use mx.init.Xavier(actor_type="in", rnd_type="gaussian", magnitude=2.0)